### ISSUE TITLE
In segment editor on a split segment it is possible to make some weird and wrong graphs

### ISSUE THREAD

**Issue by Tomhz** [JoostZ/ShowRow#55]
I don't know yet how to reproduce this. It is not a graphic issue because the resulting animation moves all over the screen. I add a screenshot.

![weird_segment_editor](https://user-images.githubusercontent.com/5354652/32807536-744cae0c-c990-11e7-8ae7-4754598b6ebe.png)

**Comment by Tomhz** [comment]
You need to be aware that i found this bug by doing some random changing of the splitted segment. I don't know if I will run into this bug in realistic situations. I fact up to now I only tested changing splitted segments without the purpose to create realistic rower movements. I will do that soon

**Comment by Tomhz** [comment]
Ok, I take back my previous comment. I did run into this bug when trying to model a realistic situation. I will increase the priority in the project

**Comment by JoostZ** [comment]
I cannot reproduce this. Can you try to find a situation where it occurs and describe the scenario? Note that there is new release which you can use to test.

**Comment by Tomhz** [comment]
In 6534.
Split Stroke Seat
Edit Stroke Seat
in segment editor, select de right subsegment
Slide T0 to the right to its maximum
Slide T3 to the left. When its value gets below 0.84 this figure show up
![rowanimation_14](https://user-images.githubusercontent.com/5354652/33097019-1dd54dbc-cf09-11e7-8f87-98d73b7d4bb0.png)

**Comment by JoostZ** [comment]
Thanks, that was what I needed. It was indeed an issue with limits. The lower limit of T3 was not updated correctly. Fixed the problem. 

I am not convinced that all limits are handled so keep an eye out for redlined boxes and other strange behavior.

**Comment by Tomhz** [comment]
Not solved yet. I can reproduce the error with a split Stroke Back Angle in 6528.
![rowanimation_17](https://user-images.githubusercontent.com/5354652/33235177-30775c80-d233-11e7-8ca4-d9a2e25758c3.png)

**Comment by JoostZ** [comment]
I cannot reproduce this. The problem with these issues is that when changing things the limits of the sliders are not always updated. I cannot see what you changed before you started changing T2. If you can try to indicate the previous steps. (Not only now but in other reports also).

**Comment by Tomhz** [comment]
In 6538:
From default, split Stroke Seat
Edit right subsegment of stroke Seat
Move T0 to max
Move T2 to min
Lower T3

![rowanimation_18](https://user-images.githubusercontent.com/5354652/33278527-1d54c114-d39c-11e7-9e84-854d3727b499.png)

**Comment by JoostZ** [comment]
Put this issue on hold since it may no longer be there because of #70 which changes the handling of changing the parameters. Issue #70 is not yet in the develop branch and so we cannot test this issue until we test #70 some more.

**Comment by Tomhz** [comment]
I have seen in in 6545, not yet in 6548. But I have no recipe yet to show it. 
More testing to be done by me.

**Comment by JoostZ** [comment]
I think this is because the handling of limits for T1, T2 and T3 is not yet correctly implemented. Please see my latest question in #70. When I know what to do I will implement it and hopefully this problem then goes away.

**Comment by Tomhz** [comment]
I cannot find what question in #70 you are referring to. 
Maybe we should leave #55 and #70 at the current status and talk about it when we are together.

**Comment by JoostZ** [comment]
I propose to close this issue. The described problem is caused by the fact that the limits for the different parameters is not correctly calculated. Therefore the UI will not prevent, e.g., T2 to be always >= T1.

However, because of #70 there will be a real overhaul of the calculation of the limits. That does not mean that #70 will solve the issue but if there are still problems after #70 the cause and probably the symptoms will be different from what is described is described here. I prefer that a new issue would be opened when the problem occurs after #70 is implemented.
